apps/crn-frontend/src/network/working-groups/api.ts

```diff
@@ -42,3 +48,27 @@ export const getWorkingGroup = async (
   }
   return resp.json();
 };
+
+export const createWorkingGroupResearchOutput = async (
```

Comment:
I don't think this is required as its a duplicate of the existing. I think because the endpoint is the same at worst we should just have the function take both types of research output